docs/integrations/code-conventions.md

```diff
@@ -713,6 +713,34 @@ timeline = IndicatorsTimeline(
 )
 ```
 
+### PollingConfiguration
+The `PollingConfiguration` is an optional object (available from Server version 6.2.0 and up). When provided to a `CommandResults` it transforms its result to a ***polling result***.
+By returning a ***polling result***, the command (or script) indicates to XSOAR that it cannot yet return the full result (likely because it's waiting for a remote process to finish execution).
+XSOAR will set the polling command to run within a number of seconds, as is set in the ***polling result***.
+
+When the time comes for the next command to run, it'll be executed.
+The polling command can return another ***polling result***, that will schedule a next run.
+
+The polling sequence will be done when either one of 3 terminating actions happen:
+
+1. ***Done*** - The polling sequence is done, indicated by an execution without any polling result.
+2. ***Error*** - The command encountered an error, indicated by an error result.
+3. ***Timeout (automatically handled)*** - The polling sequence reached the timeout, in which case a timeout error entry will be returned automatically.
+
+<img width="533" src="../doc_imgs/integrations/polling-command.png"></img>
+
+#### How to Create a Polling Command
```

Comment:
I would suggest to move this doc to separate page under Advanced Topics. And add a link in Generic Polling to point to that page (like a new feature supported since 6.2)